jasp-stats/jasp-desktop
# Add types in formula

The formula in R Syntax should add the the types of the variables (if they are changed):
`formula =  ~ contNormal + contBinom.scale`

The generation of the formula and the parsing of the formula should be then adapted for this.

Fixes https://github.com/jasp-stats/jasp-issues/issues/2990

Also the R wrappers are updated (hopefully thanks to the R Syntax project, the generation of the wrappers will be automated!)

To test it:

- Open the R Syntax, change the variables, and apply it.
- Copy the R Syntax, open a new analysis, paste it and apply it
- Duplicate analyses to check that the options can be read again in a new analysis.
